lib/vfs.js

```diff
@@ -1,65 +1,93 @@
+"use strict";
+
 var fs = require("fs");
+var utils = require("./utils");
+
+function ensureDirectory(path) {
```

Comment:
This name threw me off when I first read it. Would `ensureTrailingSlash` be clearer?